packages/shopping/src/models/email-template.model.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback4-example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class EmailTemplate extends Entity {
```

Comment:
Since there's no ID, we should extend from `Model`. See [docs](https://loopback.io/doc/en/lb4/Model.html#overview) ([archive permalink](https://github.com/strongloop/loopback-next/blob/1300169a8e76d54abdf23559fadb8afa685ba76b/docs/site/Model.md#overview))

```suggestion
export class EmailTemplate extends Model {
```